.JuliaFormatter.toml

```diff
@@ -1,2 +1,9 @@
 style = "sciml"
-whitespace_in_kwargs = false
\ No newline at end of file
+whitespace_in_kwargs = false
+always_use_return = true
+margin = 92
+indent = 2
+format_docstrings = true
+join_lines_based_on_source = true
+separate_kwargs_with_semicolon = true
+always_for_in = true
```

Comment:
what are these for?

Reply:
Most are these except the indent one just enforce some "stricter" conventions compared to SciMLStyle. Also gets rid of some conventions which I find weird, especially the return last value without an explicit return statement. Some of them need to be removed (I just copied it over from one of my other repos).